connectors/sources/s3.py

```diff
@@ -183,7 +185,7 @@ async def get_docs(self, filtering=None):
             dictionary: Document from Amazon S3.
         """
         bucket_list = self.buckets if self.buckets != ["*"] else self.get_bucket_list()
-        page_size = int(self.configuration.get("page_size", DEFAULT_PAGE_SIZE))
+        page_size = self.configuration["page_size"]
```

Comment:
Not quite sure about this change, it seems it remove the usage of `DEFAULT_PAGE_SIZE`

Reply:
no, it's still used but done directly in the base class now on init, this code is now useless.
https://github.com/elastic/connectors-python/blob/main/connectors/source.py#L124